Classes/Service/CacheTagService.php

```diff
@@ -54,7 +54,7 @@ class CacheTagService extends AbstractService implements SingletonInterface
     protected array $environments = [
         [
             self::ENVIRONMENT_TAGS => [],
-            self::ENVIRONMENT_LIFETIME => null,
+            self::ENVIRONMENT_LIFETIME => 0,
```

Comment:
Could you elaborate on what's the problem here? This is meant to mimic caching framework behavior, where `null` means "default for the configured backend" and `(int)0` actually "unlimited lifetime".

Reply:
The following lines are expecting an int value for the lifetime and return an type error:
https://github.com/netlogix/nxcachetags/blob/main/Classes/Service/CacheTagService.php#L278

This never gets set as the default livetime is null:
https://github.com/netlogix/nxcachetags/blob/main/Classes/Service/CacheTagService.php#L346

When we open an new environment we use also 0 as default value:
https://github.com/netlogix/nxcachetags/blob/main/Classes/Service/CacheTagService.php#L369